src/Codeception/Module/Filesystem.php

```diff
@@ -265,7 +265,7 @@ public function seeFileFound($filename, $path = '')
             \PHPUnit_Framework_Assert::assertFileExists($file);
             return;
         }
-        $this->fail("$filename in $path");
+        $this->fail("File \"$filename\" not found at \"$path\"");
```

Comment:
Assertion message for `seeFileFound`, 

before it that was like: `Fail: file.txt in /home/davert/Codeception`.
now it is: '`Fail: File "file.txt" not found at "/home/davert/Codeception"`.